configs/config_hg38.json

```diff
@@ -148,6 +148,11 @@
         {
             "bcftools": "/apps/bio/software/bcftools/1.9/bin/bcftools",
             "bgzip": "/apps/bio/software/htslib/htslib/bgzip"
+        },
+        "tmb_calculation":
```

Comment:
No need to do this now, but we should really containerize all tools in the pipeline...

Reply:
Agreed!